extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -190,7 +184,7 @@ public function onQueryPageUseResultsBeforeRecache( \QueryPage $queryPage, \Data
 			$handler = $this->getUnusedTemplatesHandler();
 			if ( $results instanceof \ResultWrapper ) {
 				$handler->markAsUnusedFromResults( $results );
-				$db->dataSeek( $results, 0 );	// CE-3024: reset cursor because hook caller needs the results also
+				$db->dataSeek( $results, 0 );    // CE-3024: reset cursor because hook caller needs the results also
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)